domain/src/test/java/com/thoughtworks/go/domain/materials/git/GitTestRepo.java

```diff
@@ -80,6 +80,20 @@ public GitTestRepo(File gitBundleFile, TemporaryFolder temporaryFolder) throws I
         tmpFolders.add(gitRepo);
     }
 
+    public void setColoring() {
+        createCommandLine("git").withEncoding("UTF-8").withWorkingDir(gitRepo).withArgs("config", "--global", "color.diff", "always").runOrBomb(true, "git_config");
```

Comment:
Can we set this to `--local` (implicit, unless set)? I'd really hate for this test to mess my global `.gitconfig`

Reply:
The test doesn't actually work if it's local. The test passes if the flag is local, and you remove the `--no-color` flag from `git log`

I'll see if there's a way around this...